Model training and evaluation for Mitie and Stanza

gawy opened the pull request:
- added training scripts for mitie and stanza
- single eval script

dchaplinsky commented on scripts/train_mitie_ner.py:
```diff
@@ -0,0 +1,120 @@
+        if not os.path.exists(feature_extractor_path):
+            url = 'https://dl.dropboxusercontent.com/s/87en1k1cef6wpei/total_word_feature_extractor.tokenized.400k.dat.bz2?dl=0'
+            print(f'Feature extractor file not provided or not found. \nTrying to dowload from {url}')
+            subprocess.run(['curl', url, '--output', feature_extractor_path + '.bz2'])
```
That will work for sure on nix, but ideally you should rely on `requests`/`bz2` lib of python

gawy replied:
updated in new version of code

dchaplinsky reviewed (commented):
I've mentioned few of typos in PR comments.

Some general feedback (non-mandatory, but nice-to-have):
* Could you please beatify files using `black` python lib, feel free to use 120 characters limit
* Could you please remove commented out debug prints at some stage.

On dropbox links. I think we'll switch to GIT-LFS or similar solution at some stage.

WRT to notebooks. Is it crucial part of the training pipeline or just your drafts and various tries (i.e do I need to read those)?

dchaplinsky commented on scripts/README.md:
```diff
@@ -1,5 +1,43 @@
+Vector file first must be converted to .pt format (pytorch binary format) using script from stanza `convert_pretrain.py`. More details please read [here in Stanza docs](https://stanfordnlp.github.io/stanza/word_vectors.html).
+Vector file to be converted - must start with line of format: `<word_count> <dimention [100]>`
+
+Word count can usually be obtained by `wc -l <file>`
```
Now I can see the link, sorry

gawy commented on scripts/README.md:
```diff
@@ -1,5 +1,43 @@
+
+Run: `scripts/train_stanza_ner.sh`
+
+To use custom word vectors path it with argument `--word_vec` or `-w`. **Stanza uses vectors of dimention `100`**. 
```
Do you have specific question regarding dimension or it was resolved along with your following question?

dchaplinsky replied:
I'm mentioning typo here :) dimenSions, not dimentions.

gawy commented:
- fixed typos
- cleaned up debug print 
- changed to use python requests instead of calling shell scripts
- requirements for pip are not in requirements.txt
- reformatted long lines
